gateway/utils/handlers/graphql.go

```diff
@@ -29,6 +29,10 @@ func HandleGraphQLRequest(graphql *graphql.Module, syncMan *syncman.Manager) htt
 			return
 		}
 
+		if projectConfig.ContextTime == 0 {
```

Comment:
can we set the default value on creation of project ?

Reply:
What if the project already existed? As in space cloud is using config of `v0.16.0`